From 630830476a6270e317e84229996a6bf92bd903ca Mon Sep 17 00:00:00 2001
From: Chris Bannister <c.bannister@gmail.com>
Date: Mon, 30 Nov 2015 16:26:37 -0800
Subject: [PATCH] PARQUET-396: Extend ParquetReader.Builder<T>

In AvroParquetReader.Builder extend ParquetReader.Builder<T>

Author: Chris Bannister <c.bannister@gmail.com>

Closes #294 from Zariel/PARQUET-396 and squashes the following commits:

79c1d0e [Chris Bannister] PARQUET-396: Extend ParquetReader.Builder<T>
---
 .../main/java/org/apache/parquet/avro/AvroParquetReader.java    | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/parquet-avro/src/main/java/org/apache/parquet/avro/AvroParquetReader.java b/parquet-avro/src/main/java/org/apache/parquet/avro/AvroParquetReader.java
index 0433bb9c03..a361c62fd2 100644
--- a/parquet-avro/src/main/java/org/apache/parquet/avro/AvroParquetReader.java
+++ b/parquet-avro/src/main/java/org/apache/parquet/avro/AvroParquetReader.java
@@ -70,7 +70,7 @@ public AvroParquetReader(Configuration conf, Path file, UnboundRecordFilter unbo
     super(conf, file, new AvroReadSupport<T>(), unboundRecordFilter);
   }
 
-  public static class Builder<T> extends ParquetReader.Builder {
+  public static class Builder<T> extends ParquetReader.Builder<T> {
 
     private GenericData model = null;
     private boolean enableCompatibility = true;
